packages/connect-examples/deeplink-expo/App.tsx

```diff
@@ -35,6 +35,8 @@ export const App = () => {
                 Linking.openURL(url);
             },
             deeplinkCallbackUrl: Linking.createURL('/connect'),
+            // TODO: what is the use case of 3rd party implementor setting the deep link URL?
+            // Shouldn't it be something just set in connect-deeplink package?
             // deeplinkUrl: 'https://dev.suite.sldev.cz/connect/develop/deeplink/',
```

Comment:
We need a way to tell connect-deeplink to use `https://dev.suite.sldev.cz` instead of `https://connect.trezor.io/` so it uses the development app when we are developing. 

But I think we could set it in `packages/connect-deeplink/src/data/version.ts`, and in the Connect API add something like `_use_DevDeeplinkUrl`. 

@martykan WDYT?

Reply:
What if we just used `${connectSrc}/deeplink/`
I think that would be good for consistency

Reply:
Let's do that https://github.com/trezor/trezor-suite/pull/14621/files#diff-eb7311883974581223f67cd18ddccac5c7a0efd61c687773bf6d106fa4c60ea5R167